_dashboards/admin-ui-index/forcemerge.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Force merge management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 30
+---
+
+# Force merge management
+Introduced 2.6
+{: .label .label-purple }
+
+OpenSearch Dashboards allows you to perform a [force merge]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#force_merge/) operation on indexes with **Index Management**.
+
+## Force merge indexes
+
+To perform a force merge operation on indices, perform the following steps:
+
+1. Under **Index Management**, choose **Indices**.
+
+1. Select the index you want to perform a force merge operation on.
```

Comment:
Suggested revision: "Select the index **for which you want to perform a force merge.**"